tests/test_core/test_header_generator.py

```
@@ -0,0 +1,89 @@
+"""Tests for the header generator module."""
+
+import pytest
+from sec_edgar_bulker.core.header_generator import HeaderGenerator
+from sec_edgar_bulker.core.models import Config
```

Ruff (0.8.2)

`sec_edgar_bulker.core.models.Config` imported but unused

Remove unused import: `sec_edgar_bulker.core.models.Config`

(F401)